cmd/populator-controller/populator-controller.go

```diff
@@ -3,6 +3,8 @@ package main
 import (
```

Comment:
@liranr23 please check this file

Reply:
i will try after the ovirt populator PR. note that i changed some of the lib code. the populator will run on target namespace, the secret is mounted, etc.